lhotse/kaldi.py

```diff
@@ -1,12 +1,32 @@
 import warnings
 from collections import defaultdict
 from pathlib import Path
-from typing import Any, Dict, Optional, Tuple
+from typing import Any, Dict, Optional, Tuple, NamedTuple
```

Comment:
Nit: The imported things are usually sorted alphabetically.
```suggestion
from typing import Any, Dict, NamedTuple, Optional, Tuple
```

Reply:
I will remove NamedTuple, because it isn't used.